roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -10,6 +10,12 @@
   when:
     - ceph_docker_on_openstack
 
+- name: test if the container image has disk_list function
+  command: docker run --entrypoint=stat {{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }} disk_list.sh
+  changed_when: false
+  failed_when: false
+  register: disk_list
```

Comment:
We could potentially pin this check with Jewel versions but this means handling the case where it's skipped.